libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -510,6 +511,9 @@ private List<IdentityClaim> getIdentityClaimsForBirthDateCorrelation(
                     continue;
                 }
                 addLogMessage(item, "Birthdate property is missing from VC");
+                throw new HttpResponseExceptionWithErrorBody(
```

Comment:
Is method isBirthDateEmpty() will be private ?

Reply:
Would CredentialParseException be more appropriate than HttpResponseExceptionWithErrorBody?